Source/Diagnostics/ParticleIO.cpp

```diff
@@ -112,6 +112,12 @@ MultiParticleContainer::WritePlotFile (const std::string& dir) const
                 int_flags.resize(1, 1);
             }
 
+#ifdef WARPX_QED
```

Comment:
If QED effects are enabled for a species, we add a real "optical depth component" .